engine/consensus/matching/engine.go

```diff
@@ -271,9 +266,6 @@ func (e *Engine) onReceipt(originID flow.Identifier, receipt *flow.ExecutionRece
 		return nil
 	}
 
-	log.Info().Msg("execution receipt added to mempool")
-	e.mempool.MempoolEntries(metrics.ResourceReceipt, e.receipts.Size())
```

Comment:
Removing this means, we will not track the size of the ResourceReceipts mempool anymore. 
I am ok with that, but then I think we should do it across the board for all mempools.